README.md

```diff
@@ -38,9 +38,9 @@ Review the help
 ansible-navigator --help
 ```
 
-Review current configuration (default mode)
+Review current ansible configuration (default mode)
```

Comment:
Since `-m interactive` removed from the command it can be mentioned in the heading.
`Review current ansible configuration (default mode is interacitve)`